artifacts/definitions/Linux/Sys/LogHunter.yaml

```diff
@@ -0,0 +1,46 @@
+name: Linux.Sys.LogHunter
+author: "Matt Green - @mgreen27"
+description: |
+  This artifact enables grep of Linux, MacOS and Windows logs. 
+  Parameters include SearchRegex and WhitelistRegex as regex terms.  
+  I have also included a Path exclusion regex to improve result output 
+  and automatically skip hitting notorious locations like /proc.  
+  
+  NOTE: nosymlink feature of glob is set so unexpected results may occur if 
+  targetting includes symlink files.
+  
+parameters:
+  - name: TargetFiles
+    default: '/var/log/**'
+  - name: SearchRegex
+    description: "Regex of strings to search in log line."
+    default: ' POST '
+    type: regex
+  - name: WhitelistRegex
+    description: "Regex of strings to leave out of output."
+    default:
+    type: regex
+  - name: ExcludePathRegex
+    description: "Regex of paths to exclude."
+    default: '\.journal$'
+    type: regex
+    
+sources:
+  - query: |
+      LET files = SELECT OSPath 
+        FROM glob(globs=TargetFiles,nosymlink=True,
+            recursion_callback="x=> NOT x.OSPath =~ '^/(shared|proc|snap)'")
```

Comment:
This should probably be a parameter so we can exclude any nfs mounts etc

Reply:
the only issue I had with a parameter was if the user removes all and submits a blank. We probably need to have an if there too and I wasnt sure how that works in recursion_callback